pyproject.toml

```diff
@@ -1,8 +1,4 @@
-[tool.pylint.messages_control]
-disable = "C0330, C0326"
-
-[tool.pylint.format]
-max-line-length = "88"
 
 [tool.black]
-line-length = 88
+line-length = 79
```

Comment:
Open question here w.r.t. line length. 

[Black Line Length](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length)
> Black defaults to 88 characters per line, which happens to be 10% over 80.
> If you’re using Flake8, you can bump max-line-length to 88 and forget about it. Alternatively, use Bugbear’s B950 warning instead of E501 and keep the max line length at 80 which you are probably already using.

[flake8 bugbear](https://github.com/PyCQA/flake8-bugbear#opinionated-warnings)
> B950: Line too long. This is a pragmatic equivalent of pycodestyle's E501: it considers "max-line-length" but only triggers when the value has been exceeded by more than 10%. You will no longer be forced to reformat code due to the closing parenthesis being one character too far to satisfy the linter. At the same time, if you do significantly violate the line length, you will receive a message that states what the actual limit is.

[pep8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)
> Limit all lines to a maximum of 79 characters.
> For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.
> For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the line length limit up to 99 characters, provided that comments and docstrings are still wrapped at 72 characters.

[Django coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/)
> An exception to PEP 8 is our rules on line lengths. Don’t limit lines of code to 79 characters if it means the code looks significantly uglier or is harder to read. We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult. This check is included when you run flake8. Documentation, comments, and docstrings should be wrapped at 79 characters, even though PEP 8 suggests 72.